libs/tx-builder-feature/src/utils/multicall.ts

```diff
@@ -139,8 +139,15 @@ export const handleMulticallArg = async ({
       if (data) {
         return {
           to: processedContract.address,
-          data,
-          value: Number(processValue),
+          data: isSearchArg(data)
```

Comment:
This seems odd to me. So if it's not a search arg, then it definitely has a .value & is definitely a string? 

If so, then ignore this comment.